### PR TITLE
Add new GSoC 2022 page to xmpp.org/Community

### PR DESCRIPTION
**Pull request by Echolon**
To support promoting and organizing GSoC 2022 application I created this page.

### CODE REVIEW

File: content/community/gsoc-2022.en.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+Title: XSF and Google Summer of Code 2022
+---
+
+[![](/images/logos/usshc.png)](/sponsors/us-secure-hosting-center)
+
+The XMPP Standards Foundation (XSF) is planning to participate as a hosting organisation at the [Google Summer of Code 2022](https://summerofcode.withgoogle.com/).
+If you are interested to participate as student (or simply new-commer to open-source), mentor or XMPP projects this is the page to start with!
```
Comment (wurstsalat3000) [inline content/community/gsoc-2022.en.md:8]: newcomer

Comment (wurstsalat3000) [reply content/community/gsoc-2022.en.md:8]: XMPP project

File: content/community/gsoc-2022.en.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ 
+If you can imagine to work full or part time in the summer on an open-source software project, then consider to join an XMPP real-time communication technology project and participate in the Google Summer of Code 2022!
```
Comment (wurstsalat3000) [inline content/community/gsoc-2022.en.md:10]: part time (or even full time) on an open-source project in the summer,

File: content/community/gsoc-2022.en.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+The Google Summer of Code is an international annual program in which Google awards stipends to contributors who successfully complete a free and open-source software coding project during the summer. Project ideas are listed by host organizations involved in open-source software development, though students can also propose their own project ideas. 
+
+
+### First steps
+
+If you want to participate we recommoned the following preparations:
+
+- Read the resources at [Google's website](https://summerofcode.withgoogle.com/help) and review if this is something suitable to you and your plans this year.
+- Not yet complete, but take a first look at the XMPP project details in our [XSF Wiki](https://wiki.xmpp.org/web/Google_Summer_of_Code_2022).
```
Comment (wurstsalat3000) [inline content/community/gsoc-2022.en.md:26]: - Take a first look at our XMPP project detail page in the XSF wiki (work in progress)

**Comment by Echolon** [comment]
@vanitasvitae

File: content/community/gsoc-2022.en.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+Title: XSF and Google Summer of Code 2022
+---
+
+[![](images/logos/GSoC_2022_Logo.png)](XSF and GSoC 2022 Logo)
+
+The XMPP Standards Foundation (XSF) is planning to participate as a hosting organisation at the [Google Summer of Code 2022](https://summerofcode.withgoogle.com/).
+If you are interested to participate as student (or simply newcomer to open-source), mentor or XMPP project this is the page to start with!
+ 
+If you can imagine to part time (or even full time) on an open-source software project in the summer, then consider to join an XMPP real-time communication technology project and participate in the Google Summer of Code 2022!
```
Comment (vanitasvitae) [inline content/community/gsoc-2022.en.md:10]: If you can imagine yourself part time (or even full time) working on an open-source software project during the summer, ...

File: content/community/gsoc-2022.en.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+If you want to participate we recommoned the following preparations:
+
+- Read the resources at [Google's website](https://summerofcode.withgoogle.com/help) and review if this is something suitable to you and your plans this year.
+- Take a first look at our XMPP project detail page in the [XSF wiki](https://wiki.xmpp.org/web/Google_Summer_of_Code_2022) (work in progress).
+- You can simply reach us [via web chat](https://xmpp.org/chat#converse/room?jid=gsoc@muc.xmpp.org) or review your ["Getting started with XMPP"](https://xmpp.org/getting-started/) page and reach us via our [public chat](xmpp:gsoc@muc.xmpp.org?join gsoc@muc.xmpp.org).
+
+Checkout our media channels!
+
+- [Twitter](https://twitter.com/xmpp)
+- [Fosstodon](https://fosstodon.org/@xmpp/)
```
Comment (vanitasvitae) [inline content/community/gsoc-2022.en.md:32]: I would replace Fosstodon with Mastodon/Fediverse

File: content/community/gsoc-2022.en.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+---
+Title: XSF and Google Summer of Code 2022
+---
+
+[![](images/logos/GSoC_2022_Logo.png)](XSF and GSoC 2022 Logo)
+
+The XMPP Standards Foundation (XSF) is planning to participate as a hosting organisation at the [Google Summer of Code 2022](https://summerofcode.withgoogle.com/).
```
Comment (vanitasvitae) [inline content/community/gsoc-2022.en.md:7]: ...hosting organization *in* the...

**Review by vanitasvitae — CHANGES_REQUESTED** [review]
Nice!

File: content/community/gsoc-2022.en.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
+---
+Title: XSF and Google Summer of Code 2022
+---
+
+[![](images/logos/GSoC_2022_Logo.png)](XSF and GSoC 2022 Logo)
```
Comment (vanitasvitae) [inline content/community/gsoc-2022.en.md:5]: Missing link?

File: content/community/gsoc-2022.en.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+- [Twitter](https://twitter.com/xmpp)
+- [Mastodon/Fediverse](https://fosstodon.org/@xmpp/)
+- [Youtube](https://www.youtube.com/c/XMPPStandardsFoundation)
+
+Looking forward and stay tuned!
+ –The XSF Organisation Admin
+
+### Sources
+
+- Google Summer of Code, Wikipedia contributors, Wikipedia, The Free Encyclopedia, 22 November 2021 23:03 UTC, 22 January 2022 14:31 UTC, https://en.wikipedia.org/w/index.php?title=Google_Summer_of_Code&oldid=1056637774, revision history statistics, 1056637774
```
Comment (vanitasvitae) [inline content/community/gsoc-2022.en.md:40]: I would linkify the sources to get rid of the bare URLs in the rendered page.

Comment (Echolon) [reply content/community/gsoc-2022.en.md:40]: yep, way to detailed I think

File: content/community/gsoc-2022.en.md (hunk 7 above)
Comment (Echolon) [reply content/community/gsoc-2022.en.md:5]: yes thanks, will fix

File: content/community/gsoc-2022.en.md (hunk 8 above)
Comment (Echolon) [reply content/community/gsoc-2022.en.md:40]: @vanitasvitae but do you think its legal to just link the permalink?

**Pull request merged**